README.md

```diff
@@ -18,11 +18,12 @@ If you just name it with the hostname in the inventory, it will similarly work (
 
 * `hostname_hostname`: [default: `"{{ inventory_hostname }}"`]: FQDN
 * `hostname_hostname_short`: [default: `"{{ inventory_hostname_short }}"`]: Hostname
+* `hostname_use_full`: [default: `false`]: If set to `true` `hostname` equal `FQDN`
+* `hostname_ip_address`: [default: `127.0.1.1`]: `FQDN` and `shortname` resolve to `hostname_ip_address`
```

Comment:
Can you rename it to `hostname_hostname_ip_address`?

Reply:
Yes

Reply:
Done